### PR TITLE
replace mock with teku

### PR DESCRIPTION
**Pull request by dB2510**
Replaces mock validators for node2 and node3 with teku.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -73,7 +75,33 @@ services:
       --validator-keys="/charon/node1:/charon/node1"
       --validators-proposer-default-fee-recipient="0x0000000000000000000000000000000000000000"
     volumes:
-     - .:/charon
+      - .:/charon
+
+  vc2-teku:
```
Comment (corverroos) [inline docker-compose.yml:80]: maybe extract common teku config as `x-teku-base` and then use it as a common base for the teky VCs `<<: *teku-base`?

Comment (corverroos) [reply docker-compose.yml:80]: Although, I'm not sure how to do this for common command line flags... it seems `command` needs to be duplicated every time... so not sure it is worth it...

**Review by corverroos — APPROVED** [review]
LGTM

File: docker-compose.yml (hunk 1 above)
Comment (dB2510) [reply docker-compose.yml:80]: command line flags needs to know the node for beacon node endpoint

**Pull request merged**